### DISCUSSION TITLE
NPCs turn hostile when healed

### DISCUSSION THREAD

**Discussion by Awec4** [FakeFishGames/Barotrauma#14101] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

In a multiplayer game where we had a mission to escort NPCs
Our sub was attacked by Tiger Threshers while I was outside, when I arrived back inside I found one of the mission NPCs unconscious with low oxygen/bloodloss and an infected wound, so I gave him alien blood and CPR to revive him and applied some antibiotics to the infected wound. As I was inspecting the NPC to ensure the infected wound fully healed he suddenly turned hostile and he and the others attacked me.
Presumably some part of the healing did damage e.g psychosis from the alien blood or organ damage from antibiotics
Expected behaviour: Mission NPCs on the sub should not turn hostile when being healed

### Reproduction steps

1. Start a mission to escort NPCs
2. Have an NPC be injured/knocked unconscious by creatures
3. Heal the NPC

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has now been addressed in our private repo, fix coming up in the next release. Sorry for the inconvenience!
